Changelog.md

```diff
@@ -2,10 +2,10 @@
 
 ### Added
 
--   Successfully installing an app will focus it and highlight it for
-    visibility.
--   Show help if the app list is empty (usually because all apps have been
-    filtered out).
+-   Successfully installed application will be highlighted after installation
+    finishes.
+-   Helper will be displayed if the app list is empty (usually because all apps
```

Comment:
```suggestion
-   A helper will be displayed if the app list is empty (usually because all apps
```

Reply:
In my understanding a “helper” is someone who actively helps you. But this is just a text. So I think “help text” or “help message” is better.